<launch>

<param name="~load_pred" value=0 type=bool/>
<param name="~do_record" value=1 />

<param name="~predy_file" value="" />
<param name="~predp_file" value="" />
<param name="~record_dir" value="./testrecs1" />

<param name="~ctrl_rate_hz" value=1 />
<param name="~expert_prob" value=1 />

<param name="~ang_z_gain" value=1 />
<param name="~ang_z_max" value=1 />
<param name="~ang_y_gain" value=1 />
<param name="~ang_y_max" value=1 />
<param name="~fwd_vel_gain" value=1 />
<param name="~fwd_vel_max" value=1 />



<param name="~pub_cmd_vel" value="cmd_vel_publisher" />
<param name="~pub_record" value="record_publisher" />
<param name="~pub_joy_vel" value="joy_vel_publisher" />
<param name="~pub_joy_start" value="joy_start_publisher" />
<param name="~pub_joy_stop" value="joy_stop_publisher" />
<param name="~pub_vis_feat" value="camera_publisher" />

<node name="snake_controller" pkg="slytherin_dagger" type="snake_controller_node" />


</launch>
